src/main/java/Duke.java

```diff
@@ -1,10 +1,11 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
```

Comment:
Good use of object-oriented programming concepts! Overall code was well written and clear.